src/components/RequirementHeader.vue

```diff
@@ -0,0 +1,237 @@
+          <p :style="{'font-weight': major.display ? '500' : '', 'color' : major.display ? `#${reqGroupColorMap[req.group][0]}` : ''}"  class="major-title-top">{{major.majorFN}}</p>
+          <p :style="{'color': major.display ? `#${reqGroupColorMap[req.group][0]}` : ''}" class="major-title-bottom">({{user.collegeFN}})</p>
+        </div>
+      </div>
+      <div v-if="index==2+majors.length" class="minor">
+        <div :style="{'border-bottom': minor.display ? `2px solid #${reqGroupColorMap[req.group][0]}` : ''}"
+          @click="activateMinor(id)" class="major-title" v-for="(minor, id) in minors" :key="minor.id">
+          <p :style="{'font-weight': minor.display ? '500' : '', 'color' : minor.display ? `#${reqGroupColorMap[req.group][0]}` : ''}"  class="minor-title-top">{{minor.minorFN}}</p>
+          <!-- <p :style="{'color': minor.display ? `#${reqGroupColorMap[req.group][0]}` : ''}" class="minor-title-bottom">({{user.collegeFN}})</p> Change for multiple colleges -->
+        </div>
+      </div>
+
+      <!-- progress bar settings -->
+      <div v-if="showMajorOrMinorRequirements" >
+        <div class="progress">
+          <div
+            class="progress-bar"
+            :style="{ 'background-color': `#${reqGroupColorMap[req.group][0]}`, width: `${(req.fulfilled/req.required)*100}%`}"
+            role="progressbar"
+          ></div>
+        </div>
+
+        <p class="progress-text">
+          <span class="progress-text-credits">{{ req.fulfilled }}/{{ req.required }}</span>
+          <span class="progress-text-text"> Total {{ req.type }} Inputted on Schedule</span>
+        </p>
+
+        <!--View more college requirements -->
+        <div class="row top">
+          <div class="col-1 p-0" >
+            <button :style="{ 'color': `#${reqGroupColorMap[req.group][0]}` }" class="btn" @click="toggleDetails(index)">
+              <!-- svg for dropdown icon -->
+              <img
+                v-if="req.displayDetails"
+                class="arrow arrow-up"
+                :src="require(`@/assets/images/dropup-${reqGroupColorMap[req.group][1]}.svg`)"
+                alt="dropup"
+              />
+              <img
+                v-else
+                class="arrow arrow-down"
+                :src="require(`@/assets/images/dropdown-${reqGroupColorMap[req.group][1]}.svg`)"
+                alt="dropdown"
+              />
+            </button>
+          </div>
+          <div class="col p-0 ">
+              <button
+                  class="btn req-name"
+                  :style="{ 'color': `#${reqGroupColorMap[req.group][0]}` }"
+                  @click="toggleDetails(index)">
+                  {{ (req.displayDetails) ? "Hide" : "View" }} All {{ req.group.charAt(0) + req.group.substring(1).toLowerCase() }} Requirements
+              </button>
+          </div>
+        </div>
+      </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    index: Number,
```

Comment:
we should give this a more description name

Reply:
Sure I can do `reqIndex` since it is the index of the element in the reqs array.